nemo_text_processing/text_normalization/normalize_with_audio.py

```diff
@@ -221,6 +224,7 @@ def parse_args():
     parser.add_argument(
         "--input_case", help="input capitalization", choices=["lower_cased", "cased"], default="cased", type=str
     )
+    parser.add_argument("--language", help="language", choices=["en"], default="en", type=str)
```

Comment:
please pass this arg NormalizerWithAudio() init calls (at least 2 in this script)